src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -15,17 +15,31 @@
 use Nelmio\Alice\PersisterInterface;
 
 /**
- * The Doctrine persists the fixtures into an ObjectManager
+ * Adaptor for the ObjectManager to provide a Doctrine bridge.
+ *
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ * @author Robert Schönthal
  */
 class Doctrine implements PersisterInterface
 {
-    protected $om;
+    /**
+     * @var ObjectManager
+     */
+    protected $manager;
+
+    /**
+     * @var bool
+     */
     protected $flush;
 
-    public function __construct(ObjectManager $om, $doFlush = true)
+    /**
+     * @param ObjectManager $manager
+     * @param bool          $flush
+     */
+    public function __construct(ObjectManager $manager, $flush = true)
```

Comment:
I think in v1 the point of this was that you could disable the automatic flush if you wanted to call persist/load many times and then flush them all at once. It's never been used in alice itself but was an option for users to use. Whether it still makes sense in the new code I am not sure.